packages/react-components/components/ContactCircle.tsx

```diff
@@ -25,64 +26,73 @@ const getNameInitial = (name: string) => name.charAt(0).toLocaleUpperCase()
 
 export default class ContactCircle extends React.PureComponent<Props> {
   getInitials = (): string => {
-    const { preferNameInitial, name, contact } = this.props
-    return (
-      (preferNameInitial && name && getNameInitial(name)) ||
-      (contact && getContactInitial(contact)) ||
-      (name && getNameInitial(name)) ||
-      '#'
-    )
+    const { name, contact } = this.props
+    return (contact && getContactInitial(contact)) || (name && getNameInitial(name)) || '#'
   }
 
-  getContactCircleInner = () => {
-    const { contact, size, thumbnailPath, children } = this.props
+  renderThumbnail = () => {
+    const { contact, size, thumbnailPath, name } = this.props
     const resolvedThumbnail = thumbnailPath || (contact && contact.thumbnailPath)
+    const iconSize = size || DEFAULT_ICON_SIZE
 
     if (resolvedThumbnail) {
       return (
         <Image
           source={{ uri: resolvedThumbnail }}
-          style={[style.image, { height: size, width: size, borderRadius: size / 2.0 }]}
+          style={[style.image, { height: iconSize, width: iconSize, borderRadius: iconSize / 2.0 }]}
           resizeMode={'cover'}
         />
       )
     }
 
-    // If children components (i.e. a default image) has been provided, use that
-    if (children) {
-      return children
+    // Mobile # is what default display name when contact isn't saved
```

Comment:
On both iOS and Android?

Reply:
Yeah, we assign the default display name when importing contacts to standardize it